schemas/src/registry_schemas/schemas/name_request.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/name_request",
+    "type": "object",
+    "title": "Name Request Schema",
+    "properties": {
+        "nrNumber": {
+            "type": "string"
+        },
+        "nrName": {
+            "type": "string"
+        },
+        "legalType": {
+            "type": "string",
+            "title": "The Legal Type of the business.",
+            "default": "BC",
+            "enum": [
+                "BC",
+                "CP"
+            ]
+        },
+        "requestType": {
+            "type": "string"
+        },
+        "status": {
+            "type": "string"
+        },
+        "expDate": {
+            "type": "string",
+            "format": "date"
```

Comment:
In the NR data, expiration date is datetime, although the time portion is always the same. I think though if we are getting NR data from an API call we will probably get it as datetime.

Reply:
sounds great, i will update!